techniques/flash/FLASH13.html

```diff
@@ -47,7 +47,7 @@ <h3>Applying a language just to the embedded Flash</h3>
          
             <p>This example defines the content of a Flash movie to be in the French   language. The Flash movie is embedded using <a href="https://github.com/swfobject/swfobject">SWFObject's static publishing method</a>. This means   that there are two nested object elements, the outer to target Internet   Explorer, the Inner to target other browsers. For this reason the lang   and xml:lang attributes must be added twice. </p>
          
-         <pre xml:space="preserve">&lt;object classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000"
+         <pre>&lt;object classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000"
```

Comment:
```suggestion
         <pre><code>&lt;object classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000"
```

and matching closing in https://github.com/w3c/wcag/pull/4611/files#diff-658467b219bc5b9acf64f98624b36b4976461f2b96b87fdb978f51c7e1275e05R61